highfidelity/hifi
# MS16537: Updating "Forgot Username/Forgot Password" behavior

- Fix for MS[#16537](https://highfidelity.manuscript.com/f/cases/16537/Web-views-open-behind-Log-in-Sign-up-dialog)

**TEST PLAN**

- Launch Interface
- If signed in, log out via `Menu > Log out <username>`
- Click `Menu > Login/Sign Up`
- Click `Forgot Username?` or `Forgot Password?` (test both behaviors)
- Login dialog should disappear, and browser should be set in focus
